lib/services/insights/package.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
Please update the content of License.txt in this folder(lib/services/insights) to that of MIT license. You can copy this content from [here](https://raw.githubusercontent.com/Azure/azure-sdk-for-node/master/lib/services/storageManagement2/LICENSE.txt)